extensions/amp-fx-scroll-sync/0.1/scroll-sync-scale-effect.js

```diff
@@ -25,6 +25,9 @@ export class ScrollSyncScaleEffect extends ScrollSyncEffect {
     this.endScale_ = config['end-scale'];
     this.scaleOrigin_ = config['scale-origin'];
     this.layoutBox_ = null;
+    if (this.scaleOrigin_) {
+      setStyle(this.element, 'transform-origin', this.scaleOrigin_);
```

Comment:
Is `#scaleOrigin` ever needed again? We can not set it as a property if it's not.